src/routes/safe/components/Transactions/TxList/TxCollapsed.tsx

```diff
@@ -44,6 +44,8 @@ const TxInfo = ({ info, name }: { info: AssetInfo; name?: string }) => {
       case 'SWAP_OWNER':
       case 'CHANGE_THRESHOLD':
       case 'CHANGE_IMPLEMENTATION':
+      case 'SET_GUARD' as any:
+      case 'DELETE_GUARD' as any:
```

Comment:
Do we need to add these two to settingsInfo.type in `safe-react-gateway-sdk` in order to avoid casting as any?

Reply:
If that is the case, I suggest leaving a comment to explain and create a task to fix it once `safe-react-gateway-sdk` is merged

Reply:
I saw. My suggestion was to explain the cast to `any`